inc/saischeduler.h

```diff
@@ -71,7 +71,7 @@ typedef enum _sai_scheduler_attr_t
     /**
      * @brief Scheduling algorithm weight
      *
-     * Range [1 - 100].
+     * Range [0 - 255], 0-invalid weight
```

Comment:
for better supporting tests, do we need to add capability to return what would be the max value that ASIC supports?

Reply:
Agree @r12f, let us have the capability to query the DWRR weight in the new PRs.